server.js

```diff
@@ -54,20 +54,18 @@ io.on('connection', function(socket){
     io.emit('chat message', socket.nickname + ': ' + msg);
   });
 
-  socket.on('white card', function(card){
-
+  socket.on('answer played', function(card){
     let selectingPlayer = newGame.getPlayer(socket.id)
-    // io.emit('selected white cards', {card: card, selectingPlayer: selectingPlayer} )
-
+    console.log(`An answer has been played`);
     selectedCards.push({card: card, selectingPlayer: selectingPlayer});
+    console.log(`selectedCards: ${selectedCards}`);
     let index = selectingPlayer.cards.indexOf(card);
     selectingPlayer.cards.splice(index, 1);
-    console.log(selectedCards);
-    if(selectedCards.length === players.length - 1) {
-      io.emit('selected white cards', selectedCards);
-    }
-
 
+    // If all pending answers have been played
```

Comment:
Commented out as this prevents answer cards being rendered until all players have chosen.